hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/NetworkUncheckedSubmitHandler.java

```diff
@@ -34,8 +34,11 @@
 /**
  * This class contains all workflow-related functionality regarding {@link HederaFunctionality#UNCHECKED_SUBMIT}.
  * <p>
- * This transaction type has been deprecated. Because protobufs promise backwards compatibility,
- * we cannot remove it. However, it should not be used.
+ * Unchecked submit is a mechanism to bypass ingest checks. This is a dangerous operation and will be removed
+ * in the future.
+ * Transactions that are wrapped in an unchecked submit call are unwrapped during ingest and then submitted directly.
+ * Therefore, unchecked submit is not a transaction type that is actually submitted to the network and there is no
+ * reason to implement a handler for it. This class exists for completeness but will never actually be called.
```

Comment:
```suggestion
 * reason to implement a handler for it. This class exists for completeness but should never actually be called.
```
My suggestion is subjective at best. I typically like to account for the myriad branches of possibility I haven't considered 😅

Reply:
Fixed